packages/xo-web/src/common/form/select.js

```diff
@@ -165,7 +164,7 @@ export default class Select extends React.PureComponent {
         backspaceToRemoveMessage=''
         clearable={multi || !props.required}
         closeOnSelect={!multi}
-        isLoading={!props.disabled && isEmpty(props.options)}
+        isLoading={!props.disabled && props.options == null}
```

Comment:
Prop types don't allow `null`.
And can `prop.options` ever be `null` here with a default prop `[]`?